source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -687,13 +687,14 @@ def merge_responses(responses):
 
     if not merged_response.journeys:
         # we aggregate the errors found
-
         errors = {r.error.id: r.error for r in responses if r.HasField(str('error'))}
+
+        # Only one errors field
         if len(errors) == 1:
-            merged_response.error.id = list(errors.values())[0].id
-            merged_response.error.message = list(errors.values())[0].message
-        else:
-            # we need to merge the errors
+            merged_response.error.id = errors.values()[0].id
```

Comment:
not sure about this `list` removal, it was added specifically by @kinnou02 in #2126